openasip/data/ProGe/tb/almaif/almaif-tb.vhdl.tmpl

```diff
@@ -0,0 +1,576 @@
+-- Copyright (c) 2002-2009 Tampere University.
+--
+-- This file is part of TTA-Based Codesign Environment (TCE).
+-- 
+-- Permission is hereby granted, free of charge, to any person obtaining a
+-- copy of this software and associated documentation files (the "Software"),
+-- to deal in the Software without restriction, including without limitation
+-- the rights to use, copy, modify, merge, publish, distribute, sublicense,
+-- and/or sell copies of the Software, and to permit persons to whom the
+-- Software is furnished to do so, subject to the following conditions:
+-- 
+-- The above copyright notice and this permission notice shall be included in
+-- all copies or substantial portions of the Software.
+-- 
+-- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+-- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+-- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+-- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+-- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+-- FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+-- DEALINGS IN THE SOFTWARE.
+-------------------------------------------------------------------------------
+-- Title      : testbench for TTA processor
+-- Project    : FlexDSP
+-------------------------------------------------------------------------------
+-- File       : testbench.vhdl
+-- Author     : Jaakko Sertamo  <sertamo@vlad.cs.tut.fi>
```

Comment:
Is this really still significantly based on Jaakko's ancient codes from 2001?

Reply:
No, seems that the header was copied from some other file. Based on git history seems that this file was first created in 2016 by Kati, so changed the header to that